api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -2595,6 +2597,12 @@ type HostedClusterStatus struct {
 	// +patchStrategy=merge
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
+	// Payload represents the CPU architecture type of the HostedCluster.Spec.Release.Image. The valid values are:
+	// Multi, ARM64, AMD64, or PPC64LE.
+	// +kubebuilder:validation:Enum=Multi;ARM64;AMD64;PPC64LE
+	// +optional
+	Payload string `json:"payload,omitempty"`
```

Comment:
Same comment as above

Reply:
I don't have any strong opinions. If there is a consensus on changing this, I'm happy to do so.

Reply:
Cool. I'd have questions though if the decision is to continue with this name 🙂